galleon-feature-pack/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.jboss.eap</groupId>
         <artifactId>eap-datasources-galleon-pack-parent</artifactId>
-        <version>7.4.0.GA-redhat-SNAPSHOT</version>
+        <version>8.0.0.Beta-redhat-SNAPSHOT</version>
```

Comment:
Is the version correct or should it be 8.0.0.Beta1-redhat-SNAPSHOT

Reply:
in other words, if we keep 8.0.0.Beta-redhat-SNAPSHOT, what would be the dev tag? `8.0.0.Beta1-redhat-dev` or `8.0.0.Beta-redhat-dev-00001`?

Reply:
@jmesnil , I fixed it, should have been 8.0.0.Beta1-redhat-SNAPSHOT first place. Thank-you.